### DISCUSSION TITLE
Plugin Store - Allow feature list for plugins with just one edition

### DISCUSSION THREAD

**Discussion by lindseydiloreto** [craftcms/console#107] [Plugin Store] [answered]
For plugins which have multiple editions, the Plugin Store allows them to list all of the distinctions between the two editions. Makes perfect sense, since customers would like to know which one is right for them.

However, these details are **not available** for plugins with just a single edition. This creates a misleading perception for prospective customers, and leads to an unfair marketing advantage.

Here's an example comparing two popular form plugins...

**Plugin with One Edition:**

<img width="316" alt="formie@2x" src="https://user-images.githubusercontent.com/5309692/115275085-336b4880-a0f6-11eb-9295-97bf3f5e108f.png">

**Plugin with Multiple Editions:**

<img width="655" alt="freeform@2x" src="https://user-images.githubusercontent.com/5309692/115275090-37976600-a0f6-11eb-96cc-40f6b036dc8e.png">

You tell me... From a marketing perspective, which one of those vendors is more likely to close the deal?

Would you mind making the feature list available to all plugins, regardless of whether the plugin has multiple editions? 🙏

**Comment by jalendport** [comment]
This is something I'd never considered before but I think Lindsey makes an excellent point here 👍

**Comment by brianjhanson** [comment, marked as answer]
Thanks for the feature request! We've just released an update allowing single edition plugins to add a feature list. When a feature list is present, the "Editions" tab will show (even for single editions), and users will be able to view the plugin's feature list.
